src/MicrosoftTeams.ts

```diff
@@ -1251,7 +1252,8 @@ export namespace authentication {
     if (
       hostClientType === HostClientType.desktop ||
       hostClientType === HostClientType.android ||
-      hostClientType === HostClientType.ios
+      hostClientType === HostClientType.ios ||
+      hostClientType === HostClientType.rigel
```

Comment:
Can someone confirm that we want to do the same URL conversion on Rigel as on desktop devices?  I assumed that Rigel was previously treated as a desktop device.

Reply:
@ydogandjiev

Reply:
Yes, that's the safest option; to be honest we should probably do this URL expansion in all cases.

Reply:
Great, thanks.  Yeah I agree that you might as well do it everywhere, but not a big deal either way.